scout/server/blueprints/genes/templates/genes/gene.html

```diff
@@ -107,11 +107,11 @@ <h1>
             <div class="panel panel-default">
               <div class="panel-heading">Phenotypes</div>
             {% for disease in record.disease_terms %}
```

Comment:
I don't understand here. Take for instance the ATM gene, which should be associated with many diseases, how come it has only one link to one OMIM term?


 how come it has both OMIM and ORPHA terms associated but I only see one from OMIM?

<img width="1129" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/0e05efc3-7783-46b1-bf4d-110601063067">

Reply:
The problem might also be that it only shows one link instead of all of them?

Reply:
The table heading has been updated